templates/tfengine/recipes/iam_members.hcl

```diff
@@ -56,6 +57,15 @@ schema = {
                   }
                 }
               }
+              project = {
```

Comment:
Use project_id instead of project. `project` is usually an object block in our other templates https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/examples/tfengine/devops.hcl#L38